kedro-org/kedro-viz
# Fix missing run command in metadata panel for task nodes

## Description

Resolves #2056 

## Development notes

- Update set_run_command to use KedroNode name property for constructing run_command string

## QA notes

- All tests should pass
- Copy and paste the run commands generated and kedro run should work as expected

## Checklist
